exercises/concept/secrets/.docs/instructions.md

```diff
@@ -0,0 +1,49 @@
+# Instructions
+
+In this exercise, you've been tasked with writing the software for an encryption device that works by performing transformations on data. You need a way to flexibly create complicated functions by combining simpler functions together.
+
+For each task, return a function (using a `lambda`).
+
+
+## 1. Create an adder
+
+Implement `secret_add`. It should return a function which takes one argument and adds to it the argument passed in to `secret_add`.
+
+```python
+adder = secret_add(2)
+adder(2)
+# => 4
+```

```

Comment:
```suggestion
```python
>>> adder = secret_add(2)
>>> adder(2)
4
```

```

Reply:
For better (or worse) we've decided to go the "python terminal" route with code examples.  I'll just leave the one comment - it's a small thing to clean up later.  😄